src/screens/Profile/ApprovedPosts.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  Text, View, ScrollView, StyleSheet,
+} from 'react-native';
+import firestore from '@react-native-firebase/firestore';
+import { ActivityIndicator } from 'react-native-paper';
+import PropTypes from 'prop-types';
+import auth from '@react-native-firebase/auth';
+import ForumPost from '../Forum/ForumPost';
+
+export default class ApprovedPosts extends React.Component {
```

Comment:
nit: change `... extends React.Component` to `... extends Component` and change `import React from 'react'` to `import React, { Component } from 'react'` in all files